pkgs/servers/nosql/aerospike/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchgit, autoconf, automake, libtool, openssl, zlib }:
+
+stdenv.mkDerivation rec {
+  name = "aerospike-server-${version}";
+  version = "3.14.0.1";
+
+  src = fetchgit {
```

Comment:
ah, it still does not.
```rev = "refs/tags/${version}";``` results in ```invalid character ‘/’ in name ‘aerospike-server-refs/tags/3.14.0.1-src’ ```

```rev = "${version}";``` results in ```Bad commit hash or bad reference.```

There is already an unclosed issue about too strict validation
https://github.com/NixOS/nixpkgs/issues/21620